### ISSUE TITLE
ResNet50 convergence comparison between ElasticDL and tf.keras

### ISSUE THREAD

**Issue by yhjust1** [sql-machine-learning/elasticdl#976]
Test model ResNet50 on dataset ImageNet (pick 10 categories from it)  

## Config 
**===common config===**  
dateset: ImageNet 10 classification  
train set :12850 samples  
evaluation set : 500 samples  
batch size: 128   
learning rate:0.05  
optimizer: SGD  
**===Elasticdl single worker config===**  
minibatch_size: 128  
grads_to_wait: 1  
num of worker: 1  
master_resource_request="cpu=2,memory=15048Mi,ephemeral-storage=5000Mi"  
worker_resource_request="cpu=4,memory=12048Mi,ephemeral-storage=5000Mi,gpu=1"  
**===Elasticdl two worker config===**  
minibatch_size: 64
grads_to_wait: 2  
num of worker: 2  
master_resource_request="cpu=2,memory=15048Mi,ephemeral-storage=5000Mi"  
worker_resource_request="cpu=4,memory=12048Mi,ephemeral-storage=5000Mi,gpu=1"  

## Training time  
Training time per epoch shown in table, we don't test in same environment, so the result needs further verification  

|           | worker | time |
| --------- | ------ | ---- |
| tf.keras  | -      | 70s  |
| ElasticDL | 1      | 342s |
| ElasticDL | 2      | 258s |

## Training convergence 
It seems ElasticDL with two worker have a better performance.  
![convergence comparison-single](https://user-images.githubusercontent.com/16181998/62290356-c9c78880-b493-11e9-86ff-b84a6d7dfc59.png)
![convergence comparison-two](https://user-images.githubusercontent.com/16181998/62290366-ce8c3c80-b493-11e9-8577-bc55a2cc214d.png)

## Conclusion
The ElasticDL has a slightly slower convergence speed
